index.js

```diff
@@ -0,0 +1,414 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const axios = require('axios');
+const bedrock = require('bedrock');
+const BigNumber = require('bignumber.js');
+
+const logger = require('./logger.js');
+const paymentService = require('bedrock-payment');
+
+// list of all PayPal Supported currency codes.
+const currencies = require('./currencies.js');
+
+const {PaymentStatus, Errors} = paymentService;
+
+const {config} = bedrock;
+const {BedrockError} = bedrock.util;
+
+// payment refers to a Payment.
+// You can find Payment in bedrock-web-payment:
+// @see https://github.com/digitalbazaar/bedrock-web-payment/
+
+const getConfig = () => {
+  const {api, clientId, secret} = config.paypal;
+  if(!clientId) {
+    throw new BedrockError(
+      'Missing PayPal clientId', Errors.Data);
+  }
+
+  if(!secret) {
+    throw new BedrockError(
+      'Missing PayPal secret', Errors.Data);
+  }
+
+  if(!api) {
+    throw new BedrockError(
+      'Missing PayPal api', Errors.Data);
```

Comment:
```suggestion
      'Missing PayPal API.', Errors.Data);
```

Please add punctuation to other errors.